samples/client/petstore/rust/examples/client.rs

```diff
@@ -12,32 +12,19 @@ fn main() {
     let mut core = Core::new().expect("failed to init core");
     let handle = core.handle();
 
-    let http_connector = HttpConnector::new(4, &handle);
-
-    let client = Client::configure().connector(http_connector).build(&handle);
-    let api = "http://petstore.swagger.io:80/v2";
-
     let new_pet = petstore_client::models::Pet::new("barker".to_owned(), vec![]).with_id(1337);
 
     let apicli = petstore_client::apis::client::APIClient::new(
         petstore_client::apis::configuration::Configuration::new(
             Client::configure().connector(HttpConnector::new(4, &handle)).build(&handle)));
 
-    let work = apicli.pet_api().add_pet(&new_pet)
+    let work = apicli.pet_api().AddPet(new_pet)
```

Comment:
It seems to be not the change this PR introduced, but it would be great if the naming conventions are aligned according to the [recommendations](https://aturon.github.io/style/naming.html) instead of putting `#[allow(non_snake_case)]` everywhere.

Reply:
Yeah, the python generator uses `add_pet`, though I didn't look at the Java code. Do you know how to change the Java code to get this to be the different style?

Reply:
@ahmedcharles please have a look at https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/RustClientCodegen.java#L323